homeassistant/components/nmbs/sensor.py

```diff
@@ -130,6 +130,7 @@ def device_state_attributes(self):
 
         attrs = {
             "departure": f"In {departure} minutes",
+            "departure_min": departure,
```

Comment:
Please use the entire word
```suggestion
            "departure_minutes": departure,
```